Sources/SystemInternals/Exports.swift

```diff
@@ -28,6 +30,18 @@ public var system_errno: CInt {
   get { Darwin.errno }
   set { Darwin.errno = newValue }
 }
+#elseif os(Windows)
+public var system_errno: CInt {
+  get {
+    var value: CInt = 0
+    // TODO(compnerd) handle the error?
```

Comment:
What is there to handle?

Reply:
`_get_errno` returns an `errno_t`.  It can fail in case of a PEB/TEB corruption or the parameter being invalid.